### ISSUE TITLE
Get Username of logged in user from openfire's ofuser table

### ISSUE THREAD

**Issue by mayursnarole** [maxpowel/jQuery-XMPP-plugin#30]
Hello,
How can I get the username of logged in user from ofuser table of xmpp database using jQuery-XMPP-plugin?

**Comment by golfixe** [comment]
You can get user conect? your username and nick avatar ,,, ?

**Comment by mayursnarole** [comment]
Can vCard returns username or nick avatar??

**Comment by maxpowel** [comment]
Yes, vCard can return whatever you want. In openfire you can customize your vCars

**Comment by raveesgohiel9** [comment]
Hello @maxpowel , we are trying your plugin. When I connect using your sample code. I can only see certain users online. We don't see all the online users.
Example:- I can only see users 1, user 2 online. Other users are ignored.
What could be the problem?
